hudi-common/src/test/java/org/apache/hudi/common/table/view/TestHoodieTableFileSystemView.java

```diff
@@ -306,7 +306,7 @@ protected void testInvalidLogFiles() throws Exception {
     String fileName2 =
         FSUtils.makeLogFileName(fileId, HoodieLogFile.DELTA_EXTENSION, instantTime1, 1, TEST_WRITE_TOKEN);
     // create a dummy log file mimicing cloud stores marker files
-    String fileName3 = "_DUMMY_" + fileName1.substring(1, fileName1.length());
+    String fileName3 = "_GCS_SYNCABLE_TEMPFILE_" + fileName1;
```

Comment:
Could you add this as a new one test String besides `String fileName3 = "_DUMMY_" + fileName1.substring(1, fileName1.length());`?